### PR TITLE
Add missing SELinux permission for SR-IOV

### PR DESCRIPTION
**Pull request by jean-edouard**
**What this PR does / why we need it**:
This (re-)adds the minimum set of permissions required for virt-launcher to support SR-IOV

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/virt-handler/virt_launcher.cil [hunk 1]
```diff
@@ -4,4 +4,5 @@
     (typeattributeset mcs_constrained_type (process))
     (allow process self (tun_socket (relabelfrom relabelto attach_queue)))
     (allow process hugetlbfs_t (dir (add_name create write remove_name rmdir setattr)))
-    (allow process hugetlbfs_t (file (create unlink))))
+    (allow process hugetlbfs_t (file (create unlink)))
+    (allow process sysfs_t (file (write))))
```
Comment (vladikr) [inline cmd/virt-handler/virt_launcher.cil:8]: same as in #2888

**Comment by jean-edouard** [comment]
I was finally able to run an SR-IOV test, and indeed ran into only 1 denial, the one fixed by this PR.

**Pull request merged**